crates/cdk/Cargo.toml

```diff
@@ -41,6 +41,10 @@ url.workspace = true
 utoipa = { workspace = true, optional = true }
 uuid.workspace = true
 jsonwebtoken = { workspace = true, optional = true }
+bitvec = "1.0.1"
+hex = "0.4.3"
```

Comment:
We shouldn't need to import hex, instead we can use out hex mod https://github.com/cashubtc/cdk/blob/main/crates/cashu/src/util/hex.rs